src/js/components/TableCell/TableCell.js

```diff
@@ -10,6 +10,12 @@ import { Box } from '../Box';
 import { TableContext } from '../Table/TableContext';
 import { StyledTableCell } from '../Table/StyledTable';
 
+const verticalAlignToJustify = {
+  middle: 'center',
+  top: 'start',
+  bottom: 'end',
```

Comment:
Would it make more sense to allow the Box `justify` values and allow all `justify` [options](https://v2.grommet.io/box#justify), this way we can avoid the dispatching of the values above, and the user is already familiar with the potential values? 
we can also expose just a portion of the `justify` values ('center', 'start', 'end') if it wouldn't make sense to allow all values.

Reply:
I actually started down that path but then realized that `TableCell` uses `verticalAlign` for the `<td />`, where we can't use `justify`. So, I think tying to how `<td />` works makes more sense.